templates/CRM/Campaign/Form/Task/Interview.tpl

```diff
@@ -42,7 +42,7 @@
 
     <div id='responseErrors' class = "hiddenElement messages crm-error"></div>
 
-    <div id='help'>
+    <div id='help' class='help'>
```

Comment:
I saw we just remove the id; it's redundant and the class is better.

Reply:
Done